pkg/scheduler/framework/runtime/metrics_recorder.go

```diff
@@ -14,12 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package runtime
 
 import (
 	"time"
 
 	k8smetrics "k8s.io/component-base/metrics"
+	fwk "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
```

Comment:
Please rename the variables to `fwk` instead of the import.

Reply:
sorry i don't understand this comment, could you clarify again?

Reply:
You are (possibly) using the import alias `fwk` because there are variables in the code that are named `framework`.

What I am asking is that, instead of using the import alias `fwk`, you use the alias `framework` and change conflicting variables to `fwk`.

Reply:
oh ok.....to clarify it is a [struct](https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/framework/v1alpha1/framework.go#L64)  and not a variable. 

In that case IMO changing the struct name (and not the import) may not be the best readability wise as the `framework` struct is a centrepiece of the file, and has a ton of methods that will need to be changed  i.e `func (f *fwk) getExtensionPoints(`. However let me know if you still want it changed.